taskcluster/fxci_config_taskgraph/transforms/firefoxci_artifact.py

```diff
@@ -25,8 +25,12 @@ def make_firefoxci_artifact_tasks(config, tasks):
     for task in tasks:
         # Format: { <task id>: [<artifact name>]}
         tasks_to_create = defaultdict(list)
+        include_attrs = task.pop("include-attrs", {})
+        exclude_attrs = task.pop("exclude-attrs", {})
```

Comment:
This can be done in a follow-up, but it might be nice to group `include-attrs`/`exclude-attrs` with individual index paths. E.g:
```
from-decision-indices:
    - path: <index path>
      include-attrs: [<attrs>]
      exclude-attrs: [<attrs>]
```

This would allow a bit more flexibility.